### DISCUSSION TITLE
Unknown host repo.maven.apache.org

### DISCUSSION THREAD

**Discussion by epistax1s** [community/community#53269] [Actions] [answered]
### Select Topic Area

Question

### Body

When I run the action, I get the following error:
`
Could not transfer artifact org.springframework.boot:spring-boot-starter-parent:pom:2.7.1 from/to central (https://repo.maven.apache.org/maven2): transfer failed for https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-starter-parent/2.7.1/spring-boot-starter-parent-2.7.1.pom and 'parent.relativePath' points at no local POM @ line 6, column 13: Unknown host repo.maven.apache.org: Name or service not known
`

It works locally, it has always worked on github, the file along the path `https://repo.maven.apache.org/maven2/org/springframework/boot/spring-boot-starter-parent/2.7.1/spring-boot-starter-parent-2.7.1.pom` is also available, is this the problem of github itself? When will it be fixed?

**Comment by orenf** [comment]
I saw similar failures, but the issue appears to have resolved itself. 
`Unknown host repo.maven.apache.org: Name or service not known`

**Reply by onobc** [reply]
I still see it and have been seeing it over the last 12 hrs.

**Comment by epistax1s** [comment, marked as answer]
The problem solved itself, I like it.

**Reply by orenf** [reply]
To close the loop on this, should this issue happen again, I would not know how to resolve. I haven't seen it resurface, however.

**Reply by onobc** [reply]
Seemed like a networking related blip - these things do happen. 🤷🏻

**Reply by kkajdd** [reply]
i do see that i get this  Unknown host repo.maven.apache.org today. I am behind vpn, but i get this even without vpn. Any clue to solve this?
